recovery/recovery_workflow.go

```diff
@@ -339,13 +339,15 @@ func getAllExecutionsOfType(ctx context.Context, c client.Client, workflowType s
 	var openExecutions []*commonpb.WorkflowExecution
 	var nextPageToken []byte
 	for hasMore := true; hasMore; hasMore = len(nextPageToken) > 0 {
+		zeroTime := time.Time{}
+		now := time.Now()
 		resp, err := c.ListOpenWorkflow(ctx, &workflowservice.ListOpenWorkflowExecutionsRequest{
 			Namespace:       client.DefaultNamespace,
 			MaximumPageSize: 10,
 			NextPageToken:   nextPageToken,
 			StartTimeFilter: &filterpb.StartTimeFilter{
-				EarliestTime: 0,
-				LatestTime:   time.Now().UnixNano(),
+				EarliestTime: &zeroTime,
```

Comment:
seems hard to use this way...

Reply:
I filed an issue to not require these: https://github.com/temporalio/temporal/issues/636